### PR TITLE
Added kafka service to catalog

### PR DESCRIPTION
**Pull request by rawmind0**

### CODE REVIEW

**Comment by vincent99** [comment]
@rawmind0 Can you use a vector version of the logo please: http://vtf.me/kafka.svg

(from http://svn.apache.org/repos/asf/kafka/site/logos/originals/eps/ICON%20-%20Black%20on%20White.eps)

File: templates/kafka/0/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+zk:
+  tty: true
+  image: rawmind/rancher-zk:0.0.1-1
```
Comment (cloudnautique) [inline templates/kafka/0/docker-compose.yml]: Could this be linked to the Rancher Catalog zookeeper instead of launching its own?

Comment (rawmind0) [inline templates/kafka/0/docker-compose.yml]: This version has more features. Zk cluster get reconfigured dinamically when scale up or down.

**Comment by cloudnautique** [comment]
Hello
Could you open this PR against rancher/community-catalog please? We are going to be separating out the templates provided by Rancher and the community.

**Comment by rawmind0** [comment]
Done. Resubmited PR to rancher/community-catalog

**Comment by vincent99** [comment]
This is available in [community-catalog](https://github.com/rancher/community-catalog) now.

**Pull request closed without merging**